airo-dataset-tools/test/test_pose.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+import numpy as np
+from airo_dataset_tools.pose import EulerAngles, Pose, Position
+
+
+def test_pose_simple():
+    pose = Pose(
+        position_in_meters=Position(x=1.0, y=2.0, z=3.0),
+        rotation_euler_XYZ_in_radians=EulerAngles(roll=np.pi / 4, pitch=-np.pi / 2, yaw=np.pi),
+    )
+
+    with open("pose.json", "w") as file:
+        json.dump(pose.dict(exclude_none=True), file, indent=4)
+
```

Comment:
Would it be useful to also have an example json that is parsed? Because if we now change the format by accident, the tests won't fail.

Reply:
For now I've added this to the same test because I don't know how to share the `tmp_path` directory yet.